examples/alias/main.tf

```diff
@@ -100,3 +100,52 @@ module "alias_existing" {
   }
 
 }
+
+module "sqs_events" {
+  source  = "terraform-aws-modules/sqs/aws"
+  version = "v3.3.0"
+
+  name = "${random_pet.this.id}-events"
+}
+
+module "lambda_function_event_mapping" {
```

Comment:
Drop this module call. It is a duplicate of module "lambda_function" above.

Reply:
Dropped call to this module